### ISSUE TITLE
Does plupload support to upload multiple selected files at one request?

### ISSUE THREAD

**Issue by NingerJohn** [moxiecode/plupload#1591]
## Details
I could select multiple files, however I found that plupload send multi request to upload the file to the Server.

## Question
Could plupload just send one request to upload all selected files?

**Comment by jayarjo** [comment]
No. Uploading multiple files via single request is not supported.
